ckanext/ontario_theme/templates/external/package/snippets/ontario_theme_access_level.html

```diff
@@ -9,7 +9,7 @@
       <strong>{{ _("Data Available") }}</strong>
       <p>
         {% trans license_url=_(pkg.license_url) %}
-        The data described here is available for you to use. <a href="https://www.ontario.ca/page/ontarios-open-data-directive">[Learn more]</a><br />
+        The data described here is available for you to use. <a href="https://www.ontario.ca/page/ontarios-digital-and-data-directive-2021">[Learn more]</a><br />
```

Comment:
Can you define the URL as a variable at the beginning of the html and add it into the trans tags similar to license_url? That way if the link ever changes again we only need to change it in two places instead of however many times you had to change it this time. 

You'd be able to translate just the url in the .pot and .po files and replace whats after the `href=\"` in these files with %(<variable_name>)s.

Reply:
@Alexandra-Green ready for review. 😀